pages/vuejs-development-company.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+    <!-- Hero  -->
+    <Technology.Hero iconPath="/_nuxt/assets/images/tech-stack/vue.svg" title="Vue.js Development Company"
+        subtitle="Build your own Vue.js apps with our exceptional Vue,js development services. Leverage our expertise in Fortune 500 companies to craft stable, secure & scalable software."
+        callToAction="Get Your Custom Estimate" />
+
+    <!-- Testimonials -->
+    <Testimonials />
+
+    <!-- Vue Services -->
+    <Views.Sticky.Page title="VueJS Development Services We Offer"
+        subTitle="At Acorn Globus, we offer specialized Vue.js development services to meet your unique project needs, from mobile apps to high-performance web applications."
+        :services="servicesData" />
+
+    <!-- Why Work with us ? -->
+    <Views.Sticky.Page title="Why Work With Us?"
```

Comment:
is the . convention the recommended one? as that may not be valid HTML so just confirming.